### ISSUE TITLE
How does a Polynomial Approximated Function approximate non-polynomial operators? How are the values of f1 and g1 determined? Is there any code to simulate this process?

### ISSUE THREAD

**Issue by yellow123Nike** [EfficientFHE/SmartPAF#1]
How does a Polynomial Approximated Function approximate non-polynomial operators? How are the values of f1 and g1 determined? Is there any code to simulate this process?
![image](https://github.com/EfficientFHE/SmartPAF/assets/37546857/fd025a14-bf91-474d-9691-6f920af27ada)

**Comment by yutation** [comment]
Hello,

Thank you for your question.
For the coefficients used in f1 and g1 before training, we use the values provided in the [paper](https://eprint.iacr.org/2019/1234).

**Comment by yellow123Nike** [comment]
is not 《Minimax approximation of sign function by composite polynomial for homomorphic comparison》？

**Comment by yutation** [comment]
No. Only PAF with alpha=7 uses the coefficients from the paper you mentioned. The settings with f# and g# are from the paper I mentioned in the previous reply.
